--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
	modified:   IsaacLab (modified content)
	modified:   source/unitree_rl_lab/unitree_rl_lab/assets/robots/unitree.py
	modified:   source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/IsaacLab b/IsaacLab
--- a/IsaacLab
+++ b/IsaacLab
@@ -1 +1 @@
-Subproject commit 37ddf626871758333d6ed89cf64ad702aef127d0
+Subproject commit 37ddf626871758333d6ed89cf64ad702aef127d0-dirty
diff --git a/source/unitree_rl_lab/unitree_rl_lab/assets/robots/unitree.py b/source/unitree_rl_lab/unitree_rl_lab/assets/robots/unitree.py
index b846f0d..143303f 100644
--- a/source/unitree_rl_lab/unitree_rl_lab/assets/robots/unitree.py
+++ b/source/unitree_rl_lab/unitree_rl_lab/assets/robots/unitree.py
@@ -19,8 +19,8 @@ from unitree_rl_lab.assets.robots import unitree_actuators
 
 # UNITREE_MODEL_DIR = "/home/unitree_rl_lab/unitree_model"  # Replace with the actual path to your unitree_model directory
 # UNITREE_ROS_DIR = "/home/unitree_rl_lab/unitree_ros"  # Replace with the actual path to your unitree_ros package
-UNITREE_MODEL_DIR = "/home/maverick/Humanoid/IsaacLab_New/unitree_model"
-UNITREE_ROS_DIR = "/home/maverick/Humanoid/IsaacLab_New/unitree_ros" 
+UNITREE_MODEL_DIR = "/home/IsaacLab_New/unitree_model"
+UNITREE_ROS_DIR = "/home/IsaacLab_New/unitree_ros" 
 
 @configclass
 class UnitreeArticulationCfg(ArticulationCfg):
diff --git a/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py b/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py
index dcfe946..cc51b0e 100644
--- a/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py
+++ b/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py
@@ -265,7 +265,7 @@ class ObservationsCfg:
         gait_phase = ObsTerm(func=mdp.gait_phase, params={"period": 0.8})
 
         def __post_init__(self):
-            # self.history_length = 5
+            self.history_length = 5
             self.enable_corruption = True
             self.concatenate_terms = True
 
@@ -301,8 +301,8 @@ class ObservationsCfg:
         #     clip=(-1.0, 5.0),
         # )
 
-        # def __post_init__(self):
-        #     self.history_length = 5
+        def __post_init__(self):
+            self.history_length = 5
 
     # privileged observations
     critic: CriticCfg = CriticCfg()