--- git status ---
On branch main
Your branch is up to date with 'origin/main'.

Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
  (commit or discard the untracked or modified content in submodules)
	modified:   IsaacLab (modified content)
	modified:   scripts/rsl_rl/train.py
	modified:   source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py
	modified:   source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_train_cfg.py

Untracked files:
  (use "git add <file>..." to include in what will be committed)
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_100000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_110000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_120000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_130000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_140000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_150000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_160000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_170000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_180000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_190000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_199999.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_60000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_70000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_80000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-02-24_06-00-37_symmetry/model_90000.pt
	logs/rsl_rl/unitree_h1_2_velocity_train_sym/2026-03-03_00-50-57_no_gait/

no changes added to commit (use "git add" and/or "git commit -a") 


--- git diff ---
diff --git a/IsaacLab b/IsaacLab
--- a/IsaacLab
+++ b/IsaacLab
@@ -1 +1 @@
-Subproject commit 37ddf626871758333d6ed89cf64ad702aef127d0
+Subproject commit 37ddf626871758333d6ed89cf64ad702aef127d0-dirty
diff --git a/scripts/rsl_rl/train.py b/scripts/rsl_rl/train.py
index e621154..d96ebbf 100644
--- a/scripts/rsl_rl/train.py
+++ b/scripts/rsl_rl/train.py
@@ -200,7 +200,7 @@ def main(env_cfg: ManagerBasedRLEnvCfg | DirectRLEnvCfg | DirectMARLEnvCfg, agen
         print(f"[INFO]: Loading model checkpoint from: {resume_path}")
         # load previously trained model
         runner.load(resume_path)
-
+    
     # dump the configuration into log-directory
     dump_yaml(os.path.join(log_dir, "params", "env.yaml"), env_cfg)
     dump_yaml(os.path.join(log_dir, "params", "agent.yaml"), agent_cfg)
@@ -225,6 +225,6 @@ if __name__ == "__main__":
     simulation_app.close()
 
 
-# python scripts/rsl_rl/train.py --headless --task Unitree-H1-2-Velocity-Train --num_envs 9096 --run_name H200 --max_iterations 50000 --video --video_length 200 --video_interval 2000 --distributed --max_iterations 50000 --device cuda:0,cuda:1,cuda:2,cuda:3
-# python scripts/rsl_rl/play.py --task Unitree-H1-2-Velocity-Train --num_envs 24 --checkpoint
+# python scripts/rsl_rl/train.py --headless --task Unitree-H1-2-Velocity-Train-Sym --num_envs 18192 --run_name no_gait --max_iterations 200000 --video --video_length 200 --device cuda:3
+# python scripts/rsl_rl/play.py --task Unitree-H1-2-Velocity-Train-Sym --num_envs 24 --checkpoint
 # CUDA_VISIBLE_DEVICES=0,1,2,3,4,5 
\ No newline at end of file
diff --git a/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py b/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py
index f5f47fa..bc2cc6e 100644
--- a/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py
+++ b/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_cfg.py
@@ -263,7 +263,7 @@ class ObservationsCfg:
             noise=Unoise(n_min=-1.5, n_max=1.5)
         )
         last_action = ObsTerm(func=mdp.last_action, params={"action_name": "JointPositionAction"})
-        gait_phase = ObsTerm(func=mdp.gait_phase, params={"period": 0.8})
+        # gait_phase = ObsTerm(func=mdp.gait_phase, params={"period": 0.8})
 
         def __post_init__(self):
             # self.history_length = 5
@@ -296,7 +296,7 @@ class ObservationsCfg:
             scale=0.01
         )
         last_action = ObsTerm(func=mdp.last_action, params={"action_name": "JointPositionAction"})
-        gait_phase = ObsTerm(func=mdp.gait_phase, params={"period": 0.8})
+        # gait_phase = ObsTerm(func=mdp.gait_phase, params={"period": 0.8})
         # height_scanner = ObsTerm(func=mdp.height_scan,
         #     params={"sensor_cfg": SceneEntityCfg("height_scanner")},
         #     clip=(-1.0, 5.0),
diff --git a/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_train_cfg.py b/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_train_cfg.py
index f8f1c17..d689072 100644
--- a/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_train_cfg.py
+++ b/source/unitree_rl_lab/unitree_rl_lab/tasks/locomotion/robots/h1_2/velocity_env_train_cfg.py
@@ -102,16 +102,18 @@ class RewardsTrainCfg:
     )
 
     # -- feet
-    gait = RewTerm(
-        func=mdp.feet_gait,
-        weight=1.5,
-        params={
-            "period": 0.8,
-            "offset": [0.0, 0.5],
-            "threshold": 0.55,
-            "sensor_cfg": SceneEntityCfg("contact_forces", body_names=["left_ankle_roll_link", "right_ankle_roll_link"]),
-        },
-    )
+    # gait = RewTerm(
+    #     func=mdp.feet_gait,
+    #     weight=1.5,
+    #     params={
+    #         "period": 0.8,
+    #         "offset": [0.0, 0.5],
+    #         "threshold": 0.55,
+    #         "sensor_cfg": SceneEntityCfg("contact_forces", body_names=["left_ankle_roll_link", "right_ankle_roll_link"]),
+    #     },
+    # )
+    gait = None
+    
     feet_slide = RewTerm(
         func=mdp.feet_slide,
         weight=-0.2,